--- old
+++ new
@@ -5,3 +5,1 @@
-Las mujeres aman a los hombres y en los hombres a menudo ven los niños, sin embargo rara vez son capaces de ver en los hijos, los hombres. Por eso se vuelven locas. Pero eso es otra historia.
-Aunque viva dos millones de años siempre veré a los boxeadores de Marruecos, pequeños, negros, ágiles, limpios, solos. Los giros de cintura, las esquivas, los ganchos, los yabs, la guardia francesa, el equilibrio, el miedo. Los boxeadores antes de salir al ring están luchando, mucho más incluso de lo que lucharán después. La pelea va sola, no depende del boxeador totalmente, a veces incluso se traga al boxeador. El estado natural del púgil es la soledad. Su razón de ser. Se pelea solo. Siempre. No hay otra pelea. Se gana solo. Se pierde solo. El otro no importa.
-Todos los hombres están solos o no son nada.
+Las mujeres aman a los hombres y en los hombres a menudo ven los niños, sin embargo rara vez son capaces de ver en los hijos, los hombres. 