--- old
+++ new
@@ -3,3 +3,0 @@
-Siempre es igual cuando hablo de los niños. Encima de la tumba de quien sea, sólo veo a los niños. No sé si algún día conseguiré librarme de ellos. No sé si quiero. No sé si hay algo más. Los niños son siempre los hermanos y nunca los hijos. Por eso las madres están todas locas. Encima de la tumba de los hermanos Van Gogh vi dos niños agujereados por un mismo disparo. Para un niño sólo existen las muertes de los niños. Los hombres son los que tienen que morir. Su muerte se acepta, casi ni se ve. La muerte de los niños es imposible. No hay por qué aceptarla, no existe. Luego, con los años, todas las muertes que importan son muertes de niño, muertes de hermano.
-La muerte de mi madre será una muerte de hermano y la muerte de mi padre también y hasta la muerte de mis hijos. Todo lo demás no es muerte. No es nada.
-Las mujeres aman a los hombres y en los hombres a menudo ven los niños, sin embargo rara vez son capaces de ver en los hijos, los hombres. 