--- old
+++ new
@@ -5,1 +5,1 @@
-Una y otra vez veo el mismo patrón: ante el abuso, la solución recae en la gestión privada que el abusado haga de ese conflicto, no en evidenciar y evitar futuros abusos. Creo que, de alguna forma, hemos despolitizado el sufrir abusos y 
+Una y otra vez veo el mismo patrón: ante el abuso, la solución recae en la gestión privada que el abusado haga de ese conflicto, no en evidenciar y evitar futuros abusos. Creo que, de alguna forma, hemos despolitizado el sufrir de abusos y lo hemos asimilado como parte de la vida privada. Luego la gestión del abuso depende de cada cual y no de la sociedad en su conjunto. Que una persona abusada exponga su abuso, entonces, se toma como una exposición de su vida privada, como si nos contaran un secreto. 