--- old
+++ new
@@ -2,1 +2,2 @@
-- Los bares y establecimientos similares permiten que empleados y clientes adulteren bebidas alcohólicas de mujeres 
+- Los bares y establecimientos similares permiten que empleados y clientes adulteren bebidas alcohólicas de mujeres para secuestrarlas y violarlas. Solución propuesta: enseñar a que las mujeres detecten cuando su bebida está adulterada antes de beberla. 
+- 