--- old
+++ new
@@ -3,1 +3,2 @@
-- Los reclutadores empresariales incurren en ghosting y otras dinámicas 
+- Los reclutadores empresariales incurren en ghosting y otras dinámicas abusivas hacia los candidatos. Solución propuesta: aconsejarles que sean más atractivos, que trabajen su marca personal y que no se quejen de eso en LinkedIn porque "podría ser una razón para descargarlos".
+- Empleados en una empresa acosan a mujeres jóvenes que trabajan ahí. Solución