--- old
+++ new
@@ -5,1 +5,1 @@
-Una y otra vez veo el mismo patrón: ante el abuso, la solución recae en la gestión privada que el abusado haga de ese conflicto, no en evidenciar y evitar futuros abusos. Entiendo que 
+Una y otra vez veo el mismo patrón: ante el abuso, la solución recae en la gestión privada que el abusado haga de ese conflicto, no en evidenciar y evitar futuros abusos. Creo que, de alguna forma, hemos despolitizado el sufrir abusos y 