--- old
+++ new
@@ -2,1 +2,1 @@
-Así, ese esfuerzo colectivo por sacarnos de la zona de miseria, por darnos las herramientas para la supervivencia, ha devenido, desgraciadamente, una forma radical de destrucción de clase.»
+Así, ese esfuerzo colectivo por sacarnos de la zona de miseria, por darnos las herramientas para la supervivencia, ha devenido, desgraciadamente, una forma radical de destrucción de clase.