--- old
+++ new
@@ -2,0 +2,4 @@
+«El capital cultural nos ha transformado mucho más de lo que nos transforma el capital económico. Porque puedes tener dinero y seguir siendo una mujer de barrio. Pero el capital cultural no basta con tenerlo: hay que parecerlo. Y arrasa con todo: es una forma de pensar, de moverte, de caminar, de sentarte, de sonreír o de no hacerlo; es una forma de follar y un con quién debes follar, quién te gusta y quién jamás, es un “si me escribes con faltas de ortografía en la app de ligue te bloqueo”, es un “no voy a ligar contigo porque este deje que tienes al hablar, esta cosa, estos temas no me ponen”; es la ropa, es la comida, son los gustos musicales, son los barrios por los que te mueves, es todo. El capital cultural no se adquiere, te secuestra. Y acaba contigo.
+Así, ese esfuerzo colectivo por sacarnos de la zona de miseria, por darnos las herramientas para la supervivencia, ha devenido, desgraciadamente, una forma radical de destrucción de clase.»
+
+- Brigitte Vasallo (Lenguaje Inclusivo y Exclusión de Clase) -